packages/manager/apps/dedicated/client/app/app.js

```diff
@@ -53,6 +53,7 @@ import ovhManagerCloudConnect from '@ovh-ux/manager-cloud-connect';
 import { detach as detachPreloader } from '@ovh-ux/manager-preloader';
 import ovhNotificationsSidebar from '@ovh-ux/manager-notifications-sidebar';
 import ovhManagerAccountMigration from '@ovh-ux/manager-account-migration';
+import UserAccount from '@ovh-ux/manager-user-account';
```

Comment:
```suggestion
import ovhManagerUserAccount from '@ovh-ux/manager-user-account';
```
to be consistent with other imports